src/Evolution/Executables/CurvedScalarWave/EvolveWorldtubeCurvedScalarWave.hpp

```diff
@@ -158,6 +163,28 @@ struct EvolutionMetavars {
       tmpl::list<::Events::Tags::ObserverMeshCompute<volume_dim>,
                  deriv_compute>;
 
+  template <size_t Number>
```

Comment:
If you're observing along the x and z axes, use 0 and 2 respectively rather than 1 and 2 just so the index of the coordinates matches up (even though you're not actually indexing anything)

Reply:
Oh wait, this "Number" is just an index so you can have multiple of these targets in the input file. I didn't realize this. In that case, the 0 and 2 don't make much sense actually (because they don't correspond to indices of coordinates). 0 and 1 would be better. so could you switch it again? Sorry about that...